parm/atm/obs/config/aircraft.yaml

```diff
@@ -3,15 +3,15 @@ obs space:
   obsdatain:
     engine:
       type: H5File
-      obsfile: !ENV ${DATA}/obs/${OPREFIX}aircraft.${CDATE}.nc4
+      obsfile: $(DATA)/obs/$(OPREFIX)aircraft.${{ current_cycle | to_YMDH }}.nc4
```

Comment:
not part of this PR, but we need to discuss as a group soon naming conventions for the input obs files

Reply:
Agreed, naming conventions need to be sorted out.